nesc/whip6/platforms/parts/mcu/cortex-m3/scripts/symbols2sizes.py

```diff
@@ -17,6 +17,7 @@
 KINDS = {
     'flash': 'DdGgpRrTt',
     'ram': 'CDdGgBb',
+    'gpram': 'CDdGgBb'
```

Comment:
I haven't tested it: Does it work well despite having the same value as `ram`?

Reply:
Would be nice to at least run it to test then :p

I'm not really sure about "sizes" part (build step, new target in Makefile and this change) - as far as I've seen in Makefile.main, these are not really used anywhere (correct me if I'm wrong) and even without adding those, reporting GPRAM would've worked just fine (perl script takes info from objdump anyway), so I assumed these are rather for informational / debugging purposes and added similar for GPRAM.

These values are 'kinds' of symbols that should be registered - looked up values from `man nm` ([online man here](https://linux.die.net/man/1/nm)) and IMO the same kinds fit for GPRAM.

Reply:
Please, do the tests.